examples/crud_on_store/src/lib.rs

```diff
@@ -0,0 +1,177 @@
+//! Actual schema is defined [here](https://github.com/primait/event_sourcing.rs/blob/master/src/esrs/postgres/statement.rs#L58)
+use std::convert::TryInto;
+
+use chrono::{DateTime, Utc};
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use serde_json::Value;
+use sqlx::types::Json;
+use sqlx::{Executor, Postgres};
+use uuid::Uuid;
+
+use esrs::StoreEvent;
+
+// Get all events by aggregate id: use PgStore<Manager>::by_aggregate_id function
+
+// Delete all events by aggregate id: use PgStore<Manager>::delete function
+
+// Insert event: use PgStore<Manager>::save_event function
+
+// Insert event with given event id. The table should be the aggregate name, not the aggregate_events
+pub async fn insert_event_with_given_event_id<T: Serialize + Sync>(
+    table: &str,
+    id: Uuid,
+    aggregate_id: Uuid,
+    payload: &T,
+    occurred_on: DateTime<Utc>,
+    sequence_number: i32,
+    executor: impl Executor<'_, Database = Postgres>,
+) {
+    // The path in the include_str! is the one for the given query. In your codebase you can copy the
+    // file or directly use the content
+    let name: String = format!("{}_events", table);
+    let query: String = format!(include_str!("../../../src/esrs/postgres/statements/insert.sql"), name);
```

Comment:
In the library itself I think that `statement` macro is really nice, but here it might be better to just have the queries be totally inline, to give a kind of "look there's no magic" feel to the example code?

Reply:
I thought it was better this way because in this way if i update a query you have the query replicated in the event too. Is to avoid duplication